### DISCUSSION TITLE
Authelia-Manager: A Web UI for Authelia Configuration and Management

### DISCUSSION THREAD

**Discussion by BeardedTek** [authelia/authelia#5247] [Show and tell]
![192 168 2 240_5000_ (3)](https://user-images.githubusercontent.com/93575915/232182816-f86e5323-29df-4d10-a1a7-0b797c7f05c1.png)

I have started work on a small project called [Authelia-Manager](https://github.com/beardedtek-com/authelia-manager).  It's aim is an easy to use web ui to configure and manage Authelia, similar to [nginx-proxy-manager](https://nginxproxymanager.com/) [[GitHub]](https://github.com/NginxProxyManager/nginx-proxy-manager).

## Goals
Initially, I plan to support a small subset of features, what you would typically find in a homelab or small self-hosted environment:

1. [File Based 1st Factor](https://www.authelia.com/configuration/first-factor/file/)
2. [File Based Configuration Method](https://www.authelia.com/configuration/methods/files/)
3. Generation of traefik rules for authelia integration

The reason I have these items in my initial scope is because it's my use case.

If anyone feels there should be a feaure included, please let me know.  I'll see what I can do to include it.  Also, anyone is more than welcome to take a look at my code, call me an amatuer (which I am) and give me a hand.

Hopefully it ends up something easy to use that ***"Just Works"™***.

Here's a quick UI demo.  It's very early stages, but getting there:

[![Authelia-Manager UI Demo](https://user-images.githubusercontent.com/93575915/232183406-7f43b2c6-7885-440f-9550-2995666d9f54.png)](https://youtu.be/3Ef3vwRSUsY)

**Comment by s-celles** [comment]
That was a really interesting project... but https://github.com/BeardedTek-com/authelia-manager/issues/3
